packages/e2e-tests/specs/editor/blocks/image.test.js

```diff
@@ -93,6 +97,8 @@ describe( 'Image', () => {
 
 		expect( await getEditedPostContent() ).toMatch( regex2 );
 
+		// Focus back to the img element for the right context
+		await page.click( '[aria-label="Block: Image"] img' );
```

Comment:
It didn't get unfocused, but the image caption toolbar appeared and covered the block toolbar because the image is really tiny:

<img width="432" alt="Screenshot 2021-09-02 at 12 00 36" src="https://user-images.githubusercontent.com/1451471/131824476-5fa1dad0-77ae-4825-ba1e-ae9ac2cbd62d.png">

Reply:
Ooooh interesting. I think that test isn't failing on trunk only because we're using Twenty Twenty One instead of Twenty Twenty, and the Replace button is still partially visible. Might be good to fix that even if we end up not adopting Playwright 😅 

Any reason you picked Twenty Twenty for the setup? It would be good to do the comparison based on the same theme we're using currently.

Reply:
> Any reason you picked Twenty Twenty for the setup? It would be good to do the comparison based on the same theme we're using currently.

Huh, for some reason I was sure we're using Twenty Twenty with Puppeteer! 😬 That's a great catch, will update the PR 👍

Reply:
Using TwentyTwentyOne since https://github.com/WordPress/gutenberg/pull/34089/commits/afbdd63db39a7b4e67b506cf4fe9f20993c82ee5